charts/portal/values.yaml

```diff
@@ -242,6 +242,10 @@ backend:
   userManagementPath: "/usermanagement"
   # -- Enable for using Decentral Identity Management (DIM) instead of Managed Identity Wallet (MIW) as wallet
   useDimWallet: false
+  # -- When set to false, the process worker will trigger the set sharing state ready of the bpdm, which means no manual interference on bpdm site is needed.
+  # when set to true the call to set sharing state ready is skipped. This means the bpdm site needs to set the state manually.
+  # This value should be aligned with the value set in the bpdm chart creation -> fromSharingMember -> starts-as-ready if the values are not aligned the Bpdm application checklist will fail.
+  bpdmStartAsReady: true
```

Comment:
I think it would make sense to locate this config [here](https://github.com/eclipse-tractusx/portal/blob/2414e9aad0ed25d46da721fb4a1f71b9192376a6/charts/portal/values.yaml#L811) and call it `startSharingStateAsReady`

Reply:
moved and renamed